frontend/src/metabase/api/session.ts

```diff
@@ -21,11 +23,20 @@ export const sessionApi = Api.injectEndpoints({
         body: { email },
       }),
     }),
-    getSessionProperties: builder.query<void, void>({
+    getSessionProperties: builder.query<Settings, void>({
       query: () => ({
         method: "GET",
         url: "/api/session/properties",
       }),
+      providesTags: ["session-properties"],
+      onQueryStarted: async (_, { queryFulfilled, dispatch }) => {
+        const response = await queryFulfilled;
+        if (response.data) {
+          dispatch(loadSettings(response.data));
+          // compatibility layer for legacy settings on the window object
+          MetabaseSettings.setAll(response.data);
+        }
+      },
```

Comment:
yes, we are duplicating state, but this is an elegant way of keeping it in sync. Eradicating the redux settings store would be a bigger refactor than I want to take on right now.

Reply:
actually, if anyone can figure out how to make `	getSettings()` to select from the RTK cache, that might get us most of the way there, but I banged my head on it for a bit before giving up.

Reply:
If we're ok with using MetabaseSettings as fallback, this should be almost there
```
export const getSettings: <S extends State>(state: S) => GetSettings<S> =
  createSelector(
    sessionApi.endpoints.getSessionProperties.select(),
    ({ data }) => {
      return data ? data : MetabaseSettings._settings;
    },
  );
```